lib/package.py

```diff
@@ -32,6 +33,9 @@
 
 BUILD_CACHE_DIR = os.path.join(CONF.get('default').get('work_dir'), "cache")
 
+RESULT_DIR = os.path.join(CONF.get('default').get('result_dir'),
+                          datetime.datetime.now().isoformat())
```

Comment:
This timestamp will potentially be different from the one created in `mockbuilder.py` and used as the mock's build directory name. It will then be hard to match a build directory with a corresponding results directory when debugging.
I think we could remove the `build_results_dir` attribute from here and create that path from the package name and `Mock.timestamp` in `Mock.copy_results()`, which is the only place this attribute is used.